sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -102,16 +103,33 @@ export interface AnalyzeSentimentOptions extends TextAnalyticsOperationOptions {
    * analysis around the aspects of a product or service (also known as
    * aspect-based sentiment analysis). If set to true, the returned
    * `SentenceSentiment` objects will have property `mined_opinions` containing
-   * the result of this analysis. Only available for API version v3.1-preview.1.
+   * the result of this analysis.
    * More information about the feature can be found here: https://docs.microsoft.com/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-sentiment-analysis?tabs=version-3-1#opinion-mining
    */
   includeOpinionMining?: boolean;
 }
 
+/**
+ * The different domains of PII entities that users can filter by.
+ */
+export enum PIIEntityDomainType {
+  /**
+   * See https://aka.ms/tanerpii for more information.
+   */
+  PROTECTED_HEALTH_INFORMATION = "PHI"
+}
```

Comment:
@willmtemple there was some talk yesterday about using union of strings instead but I am using enum in this case. What do you think?

Reply:
I think this is fine. My preferred pattern is to use a union of strings and then provide a convenience enum if necessary, but since there's only one in this case... maybe we should just do a string literal type?

The other thing is that I don't love the use of the word `Type` in the type name, but that's a nitpick.

Reply:
renamed `PIIEntityDomainType` to `PIIEntityDomain`.

Reply:
FYI Service plans to add more values to this type in the future.

Also @deyaaeldeen we want to keep consistency across languages and since all the others already locked the code, could you create an issue for this rename so we can discuss it for next sprint cycle and leave JS as is?
If JS feels strongly about doing the rename for this first preview, then that is ok.

Reply:
I do not think this change makes inconsistency per se. I understand Will's POV that we do not follow that naming convention in TS (we do not have that suffix in our enums, see [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md), so keeping the suffix looks odd).

Reply:
renamed `PIIEntityDomain` to `PiiEntityDomain`.

Reply:
A little confused why it doesn't cause inconsistency, also I think by your link you're saying that other TA enums don't have `type`, which I don't think makes the `Type` suffix look odd. [Here's](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/src/keysModels.ts#L29) also an instance of JS enums with `Type` suffix, but if you feel very strongly about this, you can keep it `PiiEntityDomain`

Reply:
> A little confused why it doesn't cause inconsistency

@iscai-msft why? perhaps we are thinking of different things. What is the meaning of the word `Type` here? I think you are referring to that as being an actual type i.e. a thing in the typing language. I bet @willmtemple thought the same thing. Are you using it for a different meaning?

Reply:
Just to repeat: don't have anything against `PiiEntityDomain`, am just trying to advocate we are unified in our approach (who knows, after talking we might switch this to `PiiEntityDomain` too).

When we had the meeting to talked about this, we were referring to `Type` to being a "type" of domain, not to do with a thing in the typing language

Reply:
oh I see, we were not on the same page then. Okay I will add the suffix back and update the doc string.